bedrock/firefox/templates/firefox/new/desktop/download.html

```diff
@@ -70,6 +70,17 @@
 
 {% set show_mr106_browser_promo = switch('download-firefox-mr106-promo', ['en-US', 'en-CA', 'en-GB', 'fr', 'de']) %}
 
+{% if outdated %}
+
+  {% set update_url = 'href="%s" data-cta-type="link" data-cta-text="Update to the latest version"'|safe|format(('https://support.mozilla.org/en-US/kb/update-firefox-latest-release')) %}
```

Comment:
Issue: remove `en-US` from URL path
```suggestion
  {% set update_url = 'href="%s" data-cta-type="link" data-cta-text="Update to the latest version"'|safe|format(('https://support.mozilla.org/kb/update-firefox-latest-release')) %}
```

Reply:
Should we also attach some UTM params to the SUMO link?
```suggestion
  {% set update_url = 'href="%s" data-cta-type="link" data-cta-text="Update to the latest version"'|safe|format(('https://support.mozilla.org/kb/update-firefox-latest-release' + referrals)) %}
```